packages/control-property-editor/src/panels/changes/ControlGroup.tsx

```diff
@@ -53,7 +58,9 @@ export function ControlGroup(controlGroupProps: ControlGroupProps): ReactElement
                 return (
                     <Stack.Item
                         data-testid={`${stackName}-${controlId}-${
-                            change.kind === PROPERTY_CHANGE_KIND ? change.propertyName : change.changeType
+                            change.kind === CONFIGURATION_CHANGE_KIND || change.kind === PROPERTY_CHANGE_KIND
```

Comment:
This looks like a wrong scenario. Shouldn't configuration changes never use `ControlGroup`, but use `ConfigGroup` instead?

Reply:
yes. adapted the change as well

Reply:
adapted